Modules/Microsoft365DSC/DSCResources/MSFT_IntuneTrustedRootCertificateIOS/MSFT_IntuneTrustedRootCertificateIOS.psm1

```diff
@@ -0,0 +1,698 @@
+
+        foreach ($key in ($CreateParameters.clone()).Keys)
+        {
+            if ($CreateParameters[$key].getType().Fullname -like '*CimInstance*')
+            {
+                $CreateParameters[$key] = Convert-M365DSCDRGComplexTypeToHashtable -ComplexObject $CreateParameters[$key]
+            }
+        }
+
+        $CreateParameters.add('AdditionalProperties', $AdditionalProperties)
+           
+        #region resource generator code
+        $policy = New-MgBetaDeviceManagementDeviceConfiguration @CreateParameters
+        $assignmentsHash = ConvertTo-IntunePolicyAssignment -IncludeDeviceFilter:$true -Assignments $Assignments
+
+        if ($policy.id)
+        {
+            Update-DeviceConfigurationPolicyAssignment -DeviceConfigurationPolicyId $policy.id `
+                -Targets $assignmentsHash `
+                -Repository 'deviceAppManagement/mobileAppConfigurations'
+        }
+        #endregion
+    }
+    elseif ($Ensure -eq 'Present' -and $currentInstance.Ensure -eq 'Present')
+    {
+        Write-Verbose -Message "Updating {$DisplayName}"
+        $BoundParameters.Remove('Assignments') | Out-Null
+        $UpdateParameters = ([Hashtable]$BoundParameters).clone()
+        $UpdateParameters = Rename-M365DSCCimInstanceParameter -Properties $UpdateParameters
+        $AdditionalProperties = Get-M365DSCAdditionalProperties -Properties ($UpdateParameters)
+
+        foreach ($key in $AdditionalProperties.keys)
+        {
+            if ($key -ne '@odata.type')
+            {
+                $keyName = $key.substring(0, 1).ToUpper() + $key.substring(1, $key.length - 1)
+                $UpdateParameters.remove($keyName)
+            }
+        }
+
+        if ($AdditionalProperties.ContainsKey('trustedRootCertificate')) {
+            $AdditionalProperties['trustedRootCertificate'] = [Convert]::FromBase64String($AdditionalProperties['trustedRootCertificate'])
+            Write-Verbose "trustedRootCertificate converted to bytes."
+        }
+
+        $UpdateParameters.Remove('Id') | Out-Null
+
+        foreach ($key in ($UpdateParameters.clone()).Keys)
+        {
+            if ($UpdateParameters[$key].getType().Fullname -like '*CimInstance*')
+            {
+                $UpdateParameters[$key] = Convert-M365DSCDRGComplexTypeToHashtable -ComplexObject $UpdateParameters[$key]
+            }
+        }
+        $UpdateParameters.add('AdditionalProperties', $AdditionalProperties)
+
+        #region resource generator code
+        Update-MgBetaDeviceManagementDeviceConfiguration @UpdateParameters `
+            -DeviceConfigurationId $currentInstance.Id
+        $assignmentsHash = ConvertTo-IntunePolicyAssignment -IncludeDeviceFilter:$true -Assignments $Assignments
+        Update-DeviceConfigurationPolicyAssignment -DeviceConfigurationPolicyId $currentInstance.id `
+            -Targets $assignmentsHash `
+            -Repository 'deviceAppManagement/mobileAppConfigurations'
```

Comment:
Same as above, correct repository is *'deviceManagement/deviceConfigurations'*

Reply:
ok thanks, fixed in this and in the other PR. Sorry about that.